### DISCUSSION TITLE
Unable to fix/remove/untap kde-mac/kde

### DISCUSSION THREAD

**Discussion by tskardal** [Homebrew/discussions#51] [Everyday usage] [answered]
```
❯ brew cleanup
==> We have moved our repo to KDE Invent.
Warning: GitHub repo is discontinued, archived and will no longer receive updates.
Error: In order to continue using our packages, please run the following command:
    brew untap kde-mac/kde
    brew tap kde-mac/kde https://invent.kde.org/packaging/homebrew-kde.git --force-auto-update
    `$(brew --repo kde-mac/kde)/tools/do-caveats.sh`
```

Even though I perform these commands, the error message doesn't go way. How can I get rid of this tap? (Running the untap command doesn't help).

Are there any files/folders I can delete to forcefully remove it?

**Comment by SMillerDev** [comment]
What does `brew tap` report after you do `brew untap kde-mac/kde`?

**Reply by tskardal** [reply]
Nothing unexpected. Just a list of other taps, and no error messages.

If I run untap twice, I get an error on the second execution:

```
❯ brew untap kde-mac/kde
Untapping kde-mac/kde...
Untapped 112 formulae (150 files, 9.5MB).

❯ brew untap kde-mac/kde
Error: No available tap kde-mac/kde.
```

**Reply by SMillerDev** [reply]
@yurikoles any ideas?

**Reply by tskardal** [reply]
I gave this another try, and grep'ed for the message in /usr/local/Homebrew and noticed that I got a lot of hits from a different tap, namely `haraldf/homebrew-kf5`. 

I untapped it, and voila. The error and warning is gone.
I'm not sure why I had this tap in the first place, so maybe I will experience some other issues further down the line :shrug:

**Reply by yurikoles** [reply]
Good catch `haraldf/kf5` is very old original tap name, then it was renamed to kde-mac/kde. After that we had moved away from GH to  KDE Invent (GItLab CE instance), because KDE enforces FLOSS developer tools, even main Git hosting.

**Comment by tskardal** [comment, marked as answer]
See comment https://github.com/Homebrew/discussions/discussions/51#discussioncomment-137972

**Comment by kamiqaze** [comment]
i'm having the same issue, and the answer here isn't doing anything for me. can anyone help?

**Reply by yurikoles** [reply]
You need to be more specific, what particular issue do you face and what have you tried to resolve. With commands and their outputs.

**Reply by kamiqaze** [reply]
apologies. just like op, i keep getting this prompt to any brew command, eg with cleanup:

```
> brew cleanup
==> We have moved our repo to KDE Invent.
Warning: GitHub repo is discontinued, archived and will no longer receive updates.
Error: In order to continue using our packages, please run the following command:
    brew untap kde-mac/kde
    brew tap kde-mac/kde https://invent.kde.org/packaging/homebrew-kde.git --force-auto-update
    `$(brew --repo kde-mac/kde)/tools/do-caveats.sh`
```

when i enter the above commands, the same prompt is issued again. i tried the above solution, but that doesn't work for me. please help.

**Reply by yurikoles** [reply]
I'm trying to help you, but you ignore my requests. What is printed when you invoke `brew untap kde-mac/kde`?

**Reply by yurikoles** [reply]
I don't have telepathic skills and can't help with something that happens on your Mac until you provide exact commands that you invoked and their outputs.

**Reply by kamiqaze** [reply]
> I'm trying to help you, but you ignore my requests. What is printed when you invoke `brew untap kde-mac/kde`?

like i literally said above, when i enter this command, the same prompt is repeated.

```
>brew untap kde-mac/kde
==> We have moved our repo to KDE Invent.
Warning: GitHub repo is discontinued, archived and will no longer receive updates.
Error: In order to continue using our packages, please run the following command:
    brew untap kde-mac/kde
    brew tap kde-mac/kde https://invent.kde.org/packaging/homebrew-kde.git --force-auto-update
    `$(brew --repo kde-mac/kde)/tools/do-caveats.sh`
```

**Reply by yurikoles** [reply]
Try to `rm -rf "$(brew --repo kde-mac/kde)"`

**Reply by kamiqaze** [reply]
thank you very much. this cleared it up.
